src/set_theory/cofinality.lean

```diff
@@ -97,7 +97,7 @@ namespace ordinal
 
 /-- Cofinality of an ordinal. This is the smallest cardinal of a
   subset `S` of the ordinal which is unbounded, in the sense
-  `∀ a, ∃ b ∈ S, ¬(b > a)`. It is defined for all ordinals, but
+  `∀ a, ∃ b ∈ S, a ≤ b`. It is defined for all ordinals, but
```

Comment:
I'm not familiar with the set theory library, so feel free to ignore this remark if it doesn't apply: I don't get why this change is actually better since it seems to contradict the `(λ x y, ¬ r y x)` in `strict_order.cof` and `le_cof_type, `cof_type_le`, etc.

Reply:
The relation being used here is `<`, not `>`. And since we're dealing with a linear order, `\neg(b < a)` is equivalent to `a \le b` (which is imo much clearer).